src/components/ProgressForm/InputWithQuestions.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react';
+import styles from '@/components/ProgressForm/ProgressForm.module.scss';
+
+interface inputProps {
+    name: string;
+    question: string;
+}
+
```

Comment:
Make a common file for interface

Reply:
Sure but it will increase the file count

Reply:
done